test/Benchmarks/src/Startup.enso

```diff
@@ -1,52 +0,0 @@
-from Standard.Base import all
-
-from Standard.Test import Bench
-
-polyglot java import java.lang.System as Java_System
-polyglot java import java.io.File as Java_File
-
-type Data
-    Value ~enso_bin:File ~empty_world:File ~hello_world:File
-
-    bench_empty self =
-        res = Process.run self.enso_bin.path [ "--run", self.empty_world.to_text ]
-
-    bench_hello self =
-        res = Process.run self.enso_bin.path [ "--run", self.hello_world.to_text ]
-
-
-collect_benches = Bench.build builder->
-    options = Bench.options . set_warmup (Bench.phase_conf 2 5) . set_measure (Bench.phase_conf 3 5)
-
-
-    data =
-        Data.Value enso_bin empty_file hello_file
-
-    builder.group "Startup" options group_builder->
-        group_builder.specify "empty_startup" data.bench_empty
-        group_builder.specify "hello_world_startup" data.bench_hello
-
-empty_file =
-    f = File.create_temporary_file "empty" "enso"
-    t = """
-        main = "Hello World"
-    t.write f
-    f
-
-hello_file =
-    f = File.create_temporary_file "hello" "enso"
-    t = """
-        from Standard.Base import IO
-
-        main = IO.println "Hello World"
-    t.write f
```

Comment:
Previously the benchmark created the Enso file on the fly - do you know that by default the `BenchProcessor` runs the Enso code in _"design" execution environment_ - e.g. **writing into temporary file doesn't change** its content?

To mitigate that I created `Empty_World.enso` and `Hello_World.enso` in the repository.